docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -661,19 +661,19 @@ For model_processing=*"fullaxon"*, the biophysical neuron will construct and sim
 
 Each morphology is first moved from its original coordinates in SWC to such a location that the soma is at (0, 0, 0).  Then, three rotations are applied to each morphology, in exactly the following sequence: (1) rotation around the z-axis; (2) rotation around the y-axis; (3) rotation around the x-axis.  Then, each morphology is shifted to such a location that its soma is at (x, y, z) coordinates specified in the node/node_type files.  The angles for the three rotations are also provided in the nodes/node_types files, in radians.  If a column is not provided, it is assumed that the rotation angle around that axis is 0 (that is, no rotation around that axis is applied).  The order of rotations is always the same – first around z, then around y, then around x.
 
-**morph_translate** - Optional reserved attribute, if the value is set to 0, the translation operations described above are not applied.
+**morphology** [String|Enum] - Name of the detailed morphology for a given node or node type. For name `foo`, the corresponding SWC file would be found at `$morphologies_dir/foo.swc`, where `$morphologies_dir` is specified in the [network config](#network_config).
```

Comment:
I'm still not sure that `morphology` is a good name as it can be confused with the morphology class. The description is clear, but I'd like to hear the opinion of ABI, @kaeldai, any opinion?

I'd like to have an opinion also about allowing enums, as they work completely differently from strings at parsing time. For circuits with unique morphologies enums are not providing any savings, and for circuits with few cell types it would be better to use node types. So it's only for the cases in between where it makes sense.

One comment I've already made regarding the full spec (and you also did) is that data types are missing, I appreciate that you added it here, but enums don't make sense for node_types.csv. If the enum is kept it should be clarified that it only applies to the h5 file.

Reply:
You're right about the applicability of 'Enum' in this case; it was more like " reconnaissance in force" from my side for 'Enum' applicability in general. :)